app/controllers/provider_interface/find_candidates_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ProviderInterface
+  class FindCandidatesController < ProviderInterfaceController
+    include Pagy::Backend
+    before_action :redirect_to_applications_if_no_candidate_pool_invitation
+
+    def index
+      @pagy, @candidates = pagy(
+        Pool::Candidates.for_provider(
+          providers: current_provider_user.providers,
+        ).includes(:application_forms).where(
+          application_forms: { recruitment_cycle_year: RecruitmentCycleTimetable.current_year },
+        ).order('application_forms.submitted_at'),
+      )
+    end
+
+  private
+
+    def redirect_to_applications_if_no_candidate_pool_invitation
```

Comment:
```suggestion
    def redirect_to_applications_unless_provider_opted_in
```
how do you feel about this suggestion? I was thrown by the word "invitation"